test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -156,6 +157,65 @@ public void AutoFitRankingTest()
                 Assert.True(col.Name == expectedOutputNames[col.Index]);
         }
 
+        [LightGBMFact]
+        public void AutoFitRankingCVTest()
+        {
+            string labelColumnName = "Label";
+            string groupIdColumnName = "GroupId";
+            string featuresColumnVectorNameA = "FeatureVectorA";
+            string featuresColumnVectorNameB = "FeatureVectorB";
+            int numFolds = 3;
+
+            var mlContext = new MLContext(1);
+            var dataProcessPipeline = mlContext.Transforms.Concatenate("Features", new[] { "FeatureVectorA", "FeatureVectorB" }).Append(
```

Comment:
**Suggestion, move test:** So, it looks to me this test isn't actually using AutoML? It is testing the new CV API for ranking that you're introducing on this PR (which is available for general purpose use, not only for AutoML). Thus, I think this test belongs somewhere else. Perhaps in `\test\Microsoft.ML.Functional.Tests\Validation.cs`

Reply:
@Lynx1820 Have you deleted this test? I think it was nice to have the test as it actually tested the new API, simply moving it to another place would have been ok. So I'm not sure if deleting the test altogether was intentional or not. :P

---
In reply to: [451050912](https://github.com/dotnet/machinelearning/pull/5263#discussion_r451050912) [](ancestors = 451050912)

Reply:
Yeah, I deleted it, as it seems that CV is not tested for each individual task. I was thinking I could make another PR that tests each task with CV.

Reply:
I'll resolve this on CodeFlow, as I think you've added back the test. Thanks :smile:

---
In reply to: [451970534](https://github.com/dotnet/machinelearning/pull/5263#discussion_r451970534) [](ancestors = 451970534)